### ISSUE TITLE
Check for `.markdownlint.jsonc` file by default

### ISSUE THREAD

**Issue by jjspace** [igorshubovych/markdownlint-cli#290]
The tool has the ability to parse a `jsonc` file but it doesn't look for it by default. This is one of the default configuration filenames that the vscode extension checks for so I had expected it to work.

I _think_ it should just be a matter of changing:
https://github.com/igorshubovych/markdownlint-cli/blob/1e7ba151944a8d4455fb890914bdc269bc723fda/markdownlint.js#L38
To
```js
const projectConfigFiles = ['.markdownlint.json', '.markdownlint.jsonc', '.markdownlint.yaml', '.markdownlint.yml'];
```

**Comment by DavidAnson** [comment]
That change makes sense, though I would put the new entry at the end of the list so it doesn't suddenly take precedence in cases where such a file is already present.

FYI, if you want the best compatibility with the VS Code extension, markdownlint-cli2 uses the same engine: https://github.com/DavidAnson/markdownlint-cli2
